# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/nsICanvasRenderingContextInternal.h
# Commit: b15d91e64a25
# Full Hash: b15d91e64a2589d47ad023d61dbd45135669eb60
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - NON_PREMULT TextureClient iff NON_PREMULT CanvasClient. r=handyman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78800
# ==============================================================================

diff -r 71ad2ed8e9ba -r b15d91e64a25 dom/canvas/nsICanvasRenderingContextInternal.h
--- a/dom/canvas/nsICanvasRenderingContextInternal.h	Wed Jun 10 22:21:07 2020 +0000
+++ b/dom/canvas/nsICanvasRenderingContextInternal.h	Wed Jun 10 22:21:09 2020 +0000
@@ -137,7 +137,7 @@
   virtual already_AddRefed<mozilla::gfx::SourceSurface> GetSurfaceSnapshot(
       gfxAlphaType* out_alphaType = nullptr) = 0;
 
-  virtual RefPtr<mozilla::gfx::SourceSurface> GetFrontBufferSnapshot() {
+  virtual RefPtr<mozilla::gfx::SourceSurface> GetFrontBufferSnapshot(bool) {
     return GetSurfaceSnapshot();
   }
 